cozy/ui/widgets/storages.py

```diff
@@ -0,0 +1,183 @@
+import logging
+from typing import Callable
+
+from gi.repository import Adw, Gio, GLib, GObject, Gtk
+
+from cozy.ext import inject
+from cozy.model.storage import Storage
+from cozy.view_model.storages_view_model import StoragesViewModel
+
+log = logging.getLogger("settings")
+
+
+def ask_storage_location(callback: Callable[[str], None], *junk, initial_folder: str | None = None):
```

Comment:
Normally, yeah. I use `_` everywhere to indicate that the args are junk, but where I don't, then it's because it doesn't in that places, as it would overwrite the globally defined function `_` used for string translations (alias of `gettext.gettext`). Maybe I should use `__` in in these functions, though that looks ugly.

Reply:
> as it would overwrite the globally defined function `_` used for string translations

Good point, missed it.